[2023-09-27T20:21:50.023-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.transform manual__2023-09-28T00:21:32.351221+00:00 [queued]>
[2023-09-27T20:21:50.029-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.transform manual__2023-09-28T00:21:32.351221+00:00 [queued]>
[2023-09-27T20:21:50.029-0400] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2023-09-27T20:21:50.173-0400] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): transform> on 2023-09-28 00:21:32.351221+00:00
[2023-09-27T20:21:50.180-0400] {standard_task_runner.py:57} INFO - Started process 90233 to run task
[2023-09-27T20:21:50.183-0400] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ETL_Server_Access_Log_Processing', 'transform', 'manual__2023-09-28T00:21:32.351221+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Server_Access_Log_Processing.py', '--cfg-path', '/tmp/tmpil90cbm8']
[2023-09-27T20:21:50.184-0400] {standard_task_runner.py:85} INFO - Job 23: Subtask transform
[2023-09-27T20:21:50.294-0400] {task_command.py:415} INFO - Running <TaskInstance: ETL_Server_Access_Log_Processing.transform manual__2023-09-28T00:21:32.351221+00:00 [running]> on host erosennnin-TUF-Gaming-FX505DU-FX505DU
[2023-09-27T20:21:50.543-0400] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='ETL_Server_Access_Log_Processing' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-09-28T00:21:32.351221+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-28T00:21:32.351221+00:00'
[2023-09-27T20:21:50.544-0400] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-27T20:21:50.545-0400] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'tr "[a-z]" "[A-Z]" < /home/erosennnin/Programing/Apache_Airflow/downloads/extracted.txt > /home/erosennnin/Programing/Apache_Airflow/downloads/capitalized.txt']
[2023-09-27T20:21:50.554-0400] {subprocess.py:86} INFO - Output:
[2023-09-27T20:21:50.555-0400] {subprocess.py:93} INFO - /usr/bin/bash: line 1: /home/erosennnin/Programing/Apache_Airflow/downloads/extracted.txt: No such file or directory
[2023-09-27T20:21:50.555-0400] {subprocess.py:97} INFO - Command exited with return code 1
[2023-09-27T20:21:50.561-0400] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/erosennnin/anaconda3/envs/coursera/lib/python3.11/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-27T20:21:50.563-0400] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=ETL_Server_Access_Log_Processing, task_id=transform, execution_date=20230928T002132, start_date=20230928T002150, end_date=20230928T002150
[2023-09-27T20:21:50.672-0400] {standard_task_runner.py:104} ERROR - Failed to execute job 23 for task transform (Bash command failed. The command returned a non-zero exit code 1.; 90233)
[2023-09-27T20:21:50.717-0400] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-27T20:21:50.729-0400] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
